src/components/Articles.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { useState, useEffect } from 'react';
+import { getArticles } from '../../utils/api';
+import { Link } from 'react-router-dom';
+
+const arts = [
+  {
+    author: 'grumpy19',
+    title: 'The Notorious MSG’s Unlikely Formula For Success',
+    article_id: 34,
+    topic: 'cooking',
+    created_at: '2020-11-22T11:13:00.000Z',
+    votes: 0,
+    article_img_url:
+      'https://images.pexels.com/photos/2403392/pexels-photo-2403392.jpeg?w=700&h=700',
+    comment_count: 11,
+  },
+  {
+    author: 'tickle122',
+    title: 'The battle for Node.js security has only begun',
+    article_id: 12,
+    topic: 'coding',
+    created_at: '2020-11-15T13:25:00.000Z',
+    votes: 0,
+    article_img_url:
+      'https://images.pexels.com/photos/10845119/pexels-photo-10845119.jpeg?w=700&h=700',
+    comment_count: 7,
+  },
+];
```

Comment:
This is an interesting way to handle loading so that it seems as though the fetch happens very quickly by displaying the top two articles. 
One issue is, what if those articles are one day deleted from the production database? Or if the sort order changes, then they won't be at the top of the list anymore once the fetch is complete...
It might be more future-proof to use template cards that show grey boxes for the image and title 🤔 This is an approach that is sometimes used by websites to handle when the data is still loading

Reply:
oh, sorry these two articles just for debugging, will replace with a preloader.